apps/web/src/pages/studies/[studyId]/index.tsx

```diff
@@ -194,11 +206,32 @@ export const getServerSideProps = withServerSideProps(Roles.SponsorContact, asyn
     }
   }
 
+  const { study: studyInCPMS } = await getStudyByIdFromCPMS(study.cpmsId)
+
+  if (!studyInCPMS) {
+    return {
+      redirect: {
+        destination: '/500',
```

Comment:
Are we planning on having better error feedback than this?

Reply:
Yes, there is a separate ticket for error handling/warning messages!

Reply:
@petecorriepa am I right in saying error handling comes in a later ticket?